<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonthlyTimesheetConfigurationDialog</class>
 <widget class="QWidget" name="MonthlyTimesheetConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>270</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Select month:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>comboBoxMonth</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxMonth">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Manual Selection</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Month:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="spinBoxMonth">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxYear">
          <property name="minimum">
           <number>1900</number>
          </property>
          <property name="maximum">
           <number>5000</number>
          </property>
          <property name="value">
           <number>2011</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>What Tasks to include in the report:</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QCheckBox" name="checkBoxSubTasksOnly">
          <property name="text">
           <string>Show...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelTaskName">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>(task name)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>... and subtasks</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonSelectTask">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Select Task...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>(all tasks, otherwise).</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxActiveOnly">
        <property name="text">
         <string>Only show tasks with activity</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxMonth</tabstop>
  <tabstop>spinBoxMonth</tabstop>
  <tabstop>spinBoxYear</tabstop>
  <tabstop>checkBoxSubTasksOnly</tabstop>
  <tabstop>toolButtonSelectTask</tabstop>
  <tabstop>checkBoxActiveOnly</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxSubTasksOnly</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelTaskName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxSubTasksOnly</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButtonSelectTask</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
